SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2514,6 +2520,7 @@ bool USpatialNetDriver::FindAndDispatchStartupOpsServer(const TArray<Worker_OpLi
 	}
 	else if (VirtualWorkerTranslator.IsValid() && !VirtualWorkerTranslator->IsReady())
 	{
+		GlobalStateManager->QueryTranslation();
```

Comment:
This sends a new query every tick while a successful query hasn't returned? Can we have a marker for a query being inflight to prevent dupes?

Reply:
Yeah, good idea.